app/src/features/projects/view/components/ProjectObjectives.test.tsx

```diff
@@ -36,42 +49,64 @@ describe('ProjectObjectives', () => {
 
   it('renders correctly when objectives are in multiple paragraphs', () => {
```

Comment:
this should probably say:
```suggestion
  it('renders correctly when objectives and caveats are in multiple paragraphs', () => {
```

Reply:
done

Reply:
hmm I don't see this change showing in your diff